demos/radio.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDL Radio Demo</title>
+    <script src="assets/material-design-lite.css.js" charset="utf-8"></script>
+    <style type="text/css">
+    .surface {
+      box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
+      0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+      0px 1px 3px 0px rgba(0, 0, 0, 0.12);
+
+      position: relative;
+    }
+     section {
+       margin: 48px;
+     }
+     body {
+       font-family: sans-serif;
+     }
+     button {
+       display: block;
+       margin: 16px 0;
+     }
+     .mdl-radio {
+       margin: 8px;
+     }
+     .mdl-radio span {
+       display: inline-block;
+       line-height: 24px;
+       margin-left: 4px;
+       vertical-align: top;
+     }
+    </style>
+  </head>
+  <body>
+    <main>
+      <section>
+        <label for="radio1" class="mdl-radio">
+          <div class="mdl-radio__container">
+            <div class="mdl-radio__outer-circle"></div>
+            <div class="mdl-radio__inner-circle"></div>
+            <div class="mdl-ink-ripple"></div>
+          </div>
+          <input class="mdl-radio__input" type="radio"
+                 id="radio1"
+                 checked
+                 name="myVariable">
+          <span>My Radio Button 1</span>
+        </label>
+
+        <label for="radio2" class="mdl-radio">
+          <div class="mdl-radio__container">
+            <div class="mdl-radio__outer-circle"></div>
+            <div class="mdl-radio__inner-circle"></div>
+            <div class="mdl-ink-ripple"></div>
+          </div>
+          <input class="mdl-radio__input" type="radio"
+                 id="radio2"
+                 name="myVariable">
+          <span>My Radio Button 2</span>
+        </label>
+        <button id="disable">Disable</button>
+      </section>
+    </main>
+    <script src="assets/material-design-lite.js" charset="utf-8"></script>
+    <script>
+      (function(global) {
+        'use strict';
+        var radios = document.querySelectorAll('.mdl-radio');
```

Comment:
Good point. SGTM.
On Wed, Jul 20, 2016 at 09:33 Jonathan Garbee <notifications@github.com>
wrote:

> In demos/radio.html
> <https://github.com/google/material-design-lite/pull/4566#discussion_r71524472>
> :
>
> > +            <div class="mdl-radio__inner-circle"></div>
> > +            <div class="mdl-ink-ripple"></div>
> > +          </div>
> > +          <input class="mdl-radio__input" type="radio"
> > +                 id="radio2"
> > +                 name="myVariable">
> > +          <span>My Radio Button 2</span>
> > +        </label>
> > +        <button id="disable">Disable</button>
> > +      </section>
> > +    </main>
> > +    <script src="assets/material-design-lite.js" charset="utf-8"></script>
> > +    <script>
> > +      (function(global) {
> > +        'use strict';
> > +        var radios = document.querySelectorAll('.mdl-radio');
>
> Let's not use auto init in the examples. People really shouldn't be using
> it unless they *know* what they are getting into with it. This is a good
> example of how developers can upgrade items en-mass on their projects and
> it isn't difficult to maintain if the example gets updated.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/google/material-design-lite/pull/4566/files/779475b108f75dc82c8ce4cafcf1c575e0a72f45#r71524472>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABIV9cjksXBqJbdRhSbBcM5F2m7q_-ADks5qXiOygaJpZM4JP5VQ>
> .
>